sdk/src/main/java/com/qonversion/android/sdk/UserPropertiesBuilder.kt

```diff
@@ -0,0 +1,93 @@
+package com.qonversion.android.sdk
+
+import com.qonversion.android.sdk.dto.UserProperty
+
+/**
+ * This builder class can be used to generate a map of user properties
+ * which can be then provided to [Qonversion.setUserProperties].
+ * It consumes both Qonversion defined and custom properties.
+ */
```

Comment:
I think we need to clarify that the same key of the property will overwrite the previous property value.
For example, if you call `setName` twice, only the last value is set to the dictionary.
And make attention that possible to call `setCustomUserProperty` as many times as you need. Just use different keys.